packages/optimizely-sdk/lib/plugins/odp_manager/index.browser.ts

```diff
@@ -134,6 +137,41 @@ export class BrowserOdpManager extends OdpManager {
       return new BrowserOdpManager({ disable: false, logger });
     }
 
+    let browserSegmentManager;
+    if (odpOptions.segmentsRequestHandler || odpOptions.segmentsApiTimeout) {
+      browserSegmentManager = new OdpSegmentManager(
+        new OdpConfig(),
+        new BrowserLRUCache<string, string[]>(),
```

Comment:
```suggestion
        odpOptions?.segmentsCache || new BrowserLRUCache<string, string[]>(maxSize: odpOptions.segmentsCacheSize,
          timeout: odpOptions.segmentsCacheTimeout,),
```

It's possible they want to config Cache + ApiTimeout.